BrickKit/app/build.gradle

```diff
@@ -35,21 +34,11 @@ android {
 }
 
 dependencies {
-    implementation "androidx.core:core:1.0.1"
-    implementation "androidx.fragment:fragment:1.0.0"
-    implementation "androidx.appcompat:appcompat:1.0.2"
-    implementation "androidx.recyclerview:recyclerview:1.0.0"
-    implementation "com.google.android.material:material:1.0.0"
+    implementation "androidx.core:core-ktx:1.3.2"
+    implementation "androidx.fragment:fragment-ktx:1.2.5"
+    implementation "androidx.appcompat:appcompat:1.2.0"
```

Comment:
probably doesn't matter, but we're on appcompat 1.1 in the wayfair app

Reply:
Figured it was good to see that the bricks library could be used with libraries that aren't lock step